sci-libs/cantera/cantera-3.0.0-r1.ebuild

```diff
@@ -0,0 +1,142 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+FORTRAN_NEEDED=fortran
+FORTRAN_STANDARD="77 90"
+
+inherit fortran-2 python-single-r1 scons-utils toolchain-funcs
+
+DESCRIPTION="Object-oriented tool suite for chemical kinetics, thermodynamics, and transport"
+HOMEPAGE="https://www.cantera.org"
+SRC_URI="https://github.com/Cantera/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="fortran hdf5 lapack +python test"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	lapack? ( virtual/lapack )
+	python? (
+		$(python_gen_cond_dep '
+			dev-python/numpy[${PYTHON_USEDEP}]
+			dev-python/ruamel-yaml[${PYTHON_USEDEP}]
+		')
+	)
+	dev-cpp/yaml-cpp
+	hdf5? ( sci-libs/HighFive )
+	!lapack? ( sci-libs/sundials:0= )
+	lapack? ( >=sci-libs/sundials-6.5.0:0=[lapack?] )
```

Comment:
Maybe i am missing something here but why is `lapack?` mentioned two times (just below `${PYTHON_DEPS}` aswell)
wouldn't be something like the following be enough?
```
lapack? (
	>=sci-libs/sundials-6.5.0:0=[lapack?]
	virtual/lapack
)
```

Reply:
I really missed already existent `lapack`  USE when add additional restriction for `sci-libs/sundials[lapack]` version.
I fixed it and also fixed it in additional commit for early ebuilds without revbump: there are only sort and drop of `lapack` USE duplication.